modules/apps/account/account-service/service.xml

```diff
@@ -48,6 +48,36 @@
 		<reference entity="Resource" package-path="com.liferay.portal" />
 		<reference entity="User" package-path="com.liferay.portal" />
 	</entity>
+	<entity local-service="true" name="AccountEntryOrganizationRel">
+
+		<!-- PK fields -->
+
+		<column name="accountEntryOrganizationRelId" primary="true" type="long" />
+
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+
+		<!-- Other fields -->
+
+		<column name="accountEntryId" type="long" />
+		<column name="organizationId" type="long" />
+
+		<!-- Finder methods -->
+
+		<finder name="AEI" return-type="Collection">
+			<finder-column name="accountEntryId" />
+		</finder>
```

Comment:
I think for single-column finder methods, we're supposed to use the whole column name. So in this case, the finder name would be `accountEntryId`. We do that elsewhere in this schema.